api/turing/service/experiment_service_test.go

```diff
@@ -35,9 +34,9 @@ func TestListEngines(t *testing.T) {
 	expMgr2.On("GetEngineInfo").Return(xpEngineInfo)
 
 	// Create the experiment managers map and the experiment service
-	experimentManagers := make(map[models.ExperimentEngineType]manager.ExperimentManager)
-	experimentManagers[models.ExperimentEngineTypeLitmus] = expMgr1
-	experimentManagers[models.ExperimentEngineTypeXp] = expMgr2
+	experimentManagers := make(map[string]manager.ExperimentManager)
```

Comment:
Minor: I'm wondering if we can remove the names `litmus` and `xp` from the API codebase altogether, although there is no real harm keeping them. But this can probably done later, in a separate MR to address all occurrences.